src/data/set/function.lean

```diff
@@ -810,6 +812,8 @@ open set
 
 variables {fa : α → α} {fb : β → β} {f : α → β} {g : β → γ} {s t : set α}
 
+lemma injective.inj_on_univ (h : injective f) : inj_on f univ := λ x hx y hy hxy, h hxy
```

Comment:
Given this golfs to:
```suggestion
lemma injective.inj_on_univ (h : injective f) : inj_on f univ := h.inj_on univ
```
I'm not sure a lemma that replaces a space with an underscore is worth having.

Reply:
I am not sure what you mean here

Reply:
My point is that we already have a proof of this fact as `injective.inj_on univ`. Your patch adds the lemma `injective.inj_on_univ`. The only difference between these spellings is the underscore, so I'd argue there is no point adding this new lemma.

Reply:
```suggestion
```

Reply:
I agree (although with the space is obliges to add parentheses in chains, that I hate)